apps/src/music/views/PackDialog2.tsx

```diff
@@ -0,0 +1,349 @@
+import classNames from 'classnames';
+import React, {
+  useCallback,
+  useEffect,
+  useState,
+  useRef,
+  useContext,
+} from 'react';
+import FocusLock from 'react-focus-lock';
+
+import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon';
+import SegmentedButtons from '@cdo/apps/componentLibrary/segmentedButtons';
+import Typography from '@cdo/apps/componentLibrary/typography';
+import {useAppDispatch, useAppSelector} from '@cdo/apps/util/reduxHooks';
+
+import {DEFAULT_PACK} from '../constants';
+import {AnalyticsContext} from '../context';
+import musicI18n from '../locale';
+import MusicLibrary, {SoundFolder} from '../player/MusicLibrary';
+import MusicPlayer from '../player/MusicPlayer';
+import {setPackId} from '../redux/musicRedux';
+
+import styles from './PackDialog2.module.scss';
+
+interface PackEntryProps {
+  playingPreview: string | null;
+  folder: SoundFolder;
+  folderIndex: number;
+  isSelected: boolean;
+  onSelect: (path: SoundFolder) => void;
+  onPreview: (path: string) => void;
+  mode: Mode;
+  currentFolderRefCallback: (ref: HTMLDivElement) => void;
+}
+
+const PackEntry: React.FunctionComponent<PackEntryProps> = ({
+  playingPreview,
+  folder,
+  folderIndex,
+  isSelected,
+  onSelect,
+  onPreview,
+  mode,
+  currentFolderRefCallback,
+}) => {
+  const library = MusicLibrary.getInstance();
+
+  const previewSound = folder.sounds.find(sound => sound.type === 'preview');
+  const soundPath = previewSound && folder.id + '/' + previewSound.src;
+  const isPlayingPreview = previewSound && playingPreview === soundPath;
+  const imageSrc = library?.getPackImageUrl(folder.id);
+  const imageAttributionAuthor = folder.imageAttribution?.author;
+  const imageAttributionColor = folder.imageAttribution?.color;
+  const packImageAttributionLeft = folder.imageAttribution?.position === 'left';
+
+  const onEntryClick = useCallback(() => {
+    onSelect(folder);
+
+    if (soundPath && !isPlayingPreview) {
+      onPreview(soundPath);
+    }
+  }, [folder, isPlayingPreview, onPreview, onSelect, soundPath]);
+
+  return (
+    <div
```

Comment:
Could this be a `button` element (with the `remove-button-styles` mixin) so we get the tab navigation and interaction for free?

Reply:
It does work with tab navigation now; is there more that we get from being a `button`?

Reply:
I think just potentially less code, though there might be other accessibility benefits I'm not aware of? I think general guidance is to use semantic elements where feasible, as far as I understand.